mono_repo/pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
   sdk: '>=1.24.0 <2.0.0'
 
 dependencies:
-  args: ">=0.13.7 <2.0.0"
+  args: ">=0.13.7 <1.4.0"
```

Comment:
Instead, add
```
//TODO: use non-deprecated args members – NEW ISSUE HERE
//ignore: deprecated_member_use
```
to the offending lines?

Reply:
I agree with the TODO:

But wouldn't the ignore cause problem in case of a `>=1.4.0 <2.0.0` having a breaking change on this ?

Reply:
The fix was in fact minor. Might as well do it.